aiohttp/connector.py

```diff
@@ -1126,3 +1127,55 @@ def path(self) -> str:
             raise ClientConnectorError(req.connection_key, exc) from exc
 
         return cast(ResponseHandler, proto)
+
+
+class NamedPipeConnector(BaseConnector):
+    """Named pipe connector.
```

Comment:
According to PEP257, there must be one empty line after a title line.
```suggestion
    """Named pipe connector.

```